zulip/zulip/__init__.py

```diff
@@ -864,6 +864,20 @@ def get_realm_emoji(self):
             method='GET'
         )
 
+    def upload_custom_emoji(self, emoji_name, file):
+        # type: (str, IO[Any]) -> Dict[str, Any]
+        '''
+            Example usage:
+
+            >>> client.upload_custom_emoji(emoji_name, file)
```

Comment:
I think the example call to the function here could be more explicit. We should just make it clear that `file` is a file object and not just a path.

Also, it is probably better to use another name such as `file_obj`, the name `file` is a reserved keyword in Python! :)

Reply:
I changed it, as you suggest :)